lavalink/models.py

```diff
@@ -35,6 +35,7 @@ def build(cls, track, requester, **kwargs):
             new_track.stream = track['info']['isStream']
             new_track.title = track['info']['title']
             new_track.uri = track['info']['uri']
+            new_track.extra = extra or {}
```

Comment:
I don't get why is this here and not like it was. Seems like over-complicating

Reply:
Since I defaulted it as `None` it would either be `{}` or if it's set then the actual dict.

Reply:
or use `__init__` and have self.extra = extra
and if i'm not mistaken kwargs returns empty dict

Reply:
It does, but we're not using kwarg here

Reply:
So this is fine...?

Reply:
I should think so